src/main/kotlin/blackjack/view/Output.kt

```diff
@@ -0,0 +1,23 @@
+package blackjack.view
+
+import blackjack.model.Player
+
+object Output {
+    fun players(players: List<Player>) {
+        println("${players.joinToString(",") { "${it.name}" }}에게 2장의 나누었습니다.")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        println("${players.joinToString(",") { it.name }}에게 2장의 나누었습니다.")
```